addons/hr_skills/static/src/fields/skills_one2many/skills_one2many.xml

```diff
@@ -9,7 +9,7 @@
         </xpath>
         <xpath expr="//table" position="before">
             <div name="skills_header" class="text-uppercase fw-bolder small ms-3" style="margin-top: 2rem; box-shadow: 0 1px 0 #e6e6e6">
-                Skills <a t-on-click="openSkillsReport" class="fa fa-line-chart ms-1 cursor-pointer"/>
+                Skills <a t-on-click="openSkillsReport" class="fa fa-line-chart float-end me-3 cursor-pointer"> Timeline </a>
```

Comment:
I think it's the font-awesome classes that change the font, maybe try something like this?
```suggestion
                Skills
                <a t-on-click="openSkillsReport" class="float-end me-3 cursor-pointer">
                    <span class="fa fa-line-chart me-1"/>
                     Timeline
                 </a>
```